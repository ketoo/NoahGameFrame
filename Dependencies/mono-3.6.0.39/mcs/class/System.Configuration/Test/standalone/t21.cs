using System;
using System.Configuration;

class T1
{
	static void Main(string[] args)
	{
		try {
			Console.WriteLine ("DefaultProvider = {0}", ProtectedConfiguration.DefaultProvider);
			RsaProtectedConfigurationProvider rsa = (RsaProtectedConfigurationProvider)ProtectedConfiguration.Providers [ProtectedConfiguration.DefaultProvider];
			Console.WriteLine (rsa.Name);

			Console.WriteLine ("cspProviderName = '{0}'", rsa.CspProviderName == null ? "(null)" : rsa.CspProviderName);
			Console.WriteLine ("keyContainerName = '{0}'", rsa.KeyContainerName == null ? "(null)" : rsa.KeyContainerName);
			Console.WriteLine ("useMachineContainer = '{0}'", rsa.UseMachineContainer);
			Console.WriteLine ("useOAEP = '{0}'", rsa.UseOAEP);
		}
		catch (Exception e)
		{
			Console.WriteLine ("Exception raised: {0}", e);
		}
	}
}
