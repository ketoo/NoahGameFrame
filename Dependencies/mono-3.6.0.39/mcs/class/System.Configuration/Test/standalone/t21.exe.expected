DefaultProvider = MyRsaProtectedConfigurationProvider
MyRsaProtectedConfigurationProvider
cspProviderName = '(null)'
keyContainerName = 'MyKeys'
useMachineContainer = 'True'
useOAEP = 'True'
